scipy/stats/_sensitivity_analysis.py

```diff
@@ -612,6 +612,15 @@ def funcAB(AB):
                 f_AB.shape == f_A.shape or f_AB.shape[-1] % n != 0:
             raise ValueError(message)
 
+    # Normalization by mean
+    # Sobol’, I. and Levitan, Y. L. (1999). On the use of variance reducing
+    # multipliers in monte carlo computations of a global sensitivity index.
+    # Computer Physics Communications, 117(1) :52–61.
+    mean = np.mean([f_A, f_B], axis=(0, -1))
```

Comment:
That is exactly what I had in mind. It comes as close a possible to Janond-Monod's estimator, while keeping the orginal idea from @andreasaltelli.